check_vmware_api.pl

```diff
@@ -1219,20 +1220,20 @@ sub datastore_volumes_info
 			if ($store->summary->accessible)
 			{
 				$store->RefreshDatastoreStorageInfo() if ($store->can("RefreshDatastoreStorageInfo") && exists($store->info->{timestamp}) && $defperfargs->{timeshift} && (time() - str2time($store->info->timestamp) > $defperfargs->{timeshift}));
-				my $value1 = simplify_number(convert_number($store->summary->freeSpace) / 1024 / 1024);
```

Comment:
Yes, it seems to be used in other places. From what I understand, `convert_number` removes extra whitespace and leading commas while `simplify_number` rounds floats.
I'm not exactly sure why `convert_number` needs to call `split(/,/` but I did not want to touch that part of the code.